public/script.js

```diff
@@ -14,22 +12,19 @@ $(document).ready(function() {
   });
 
   $("body").on("click", '.send', function() {
-    var $input = $(this).parents('.write').find('.textarea');
+    const $input = $(this).parents('.write').find('.textarea');
```

Comment:
You might want to remove t his `$` symbol from name of variable as this is useless.